src/patterns/help-users-to/cookies-settings/examples/cookie-settings-page/index.njk

```diff
@@ -45,10 +45,12 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
 {% from "components/cookies-banner/_macro.njk" import onsCookiesBanner %}
 {{
     onsCookiesBanner({
-        "statementTitle": 'Tell us whether you accept cookies',
-        "statementText": 'We use <a href="#0">cookies to collect information</a> about how you use census.gov.uk. We use this information to make the website work as well as possible and improve our services.',
-        "confirmationText": 'You’ve accepted all cookies. You can <a href="#0">change your cookie preferences</a> at any time.',
-        "secondaryButtonUrl": '#'
+        "statementTitle": 'Cookies on ons.gov.uk',
```

Comment:
Service name should be separate param.
```suggestion
        "statementTitle": 'Cookies on',
```
Duplication in example:
<img width="672" alt="Screenshot 2022-07-25 at 12 18 19" src="https://user-images.githubusercontent.com/43346934/180765935-6e6c700b-7e85-4763-86e8-b04e099f1af8.png">